app/controllers/laa_references_controller.rb

```diff
@@ -58,8 +55,8 @@ def link_attempt_params
     laa_reference_params[:link_attempt].merge(no_maat_id: no_maat_id?)
   end
 
-  def defendant_identifier
-    @defendant_identifier = laa_reference_params[:id] || link_attempt_params[:defendant_identifier]
+  def defendant_asn_or_nino
+    @defendant_asn_or_nino = laa_reference_params[:id] || link_attempt_params[:defendant_asn_or_nino]
```

Comment:
Does it break anything to use `||=` rather than '=' ? I am not sure of impact however

Reply:
👍 now using ||=